pkg/controller/clone-controller.go

```diff
@@ -363,18 +161,72 @@ func (c *CloneController) processPvcItem(pvc *v1.PersistentVolumeClaim) error {
 	if !checkIfLabelExists(pvc, common.CDILabelKey, common.CDILabelValue) {
 		lab = map[string]string{common.CDILabelKey: common.CDILabelValue}
 	}
-	pvc, err = updatePVC(c.clientset, pvc, anno, lab)
+	pvc, err = updatePVC(cc.clientset, pvc, anno, lab)
 	if err != nil {
 		return errors.WithMessage(err, "could not update pvc %q annotation and/or label")
 	}
 	return nil
 }
 
-// forget the passed-in key for this event and optionally log a message.
-func (c *CloneController) forgetKey(key interface{}, msg string) bool {
-	if len(msg) > 0 {
-		glog.V(3).Info(msg)
+func (c *Controller) initializeExpectations(pvcKey string) error {
+	return c.podExpectations.SetExpectations(pvcKey, 0, 0)
+}
+
+func (c *Controller) expectPodCreate(pvcKey string) {
```

Comment:
so just a though now that we noticed this... rather than have multiple functions with the same name that does different things; maybe we could/should implement these in the general controller still but give them meaningful names.  What I'm getting at is something like adding two functions in controller.go:

> func(*c controller)raisePodExpect()
> func(*c controller)createPodExpect()

That would have eliminated this sort of confusion, IMO if the functoin does different things it shouldn't have the same name :)